railties/lib/rails/code_statistics.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+# :markup: markdown
```

Comment:
I think we only want to change docs to markdown entire frameworks at a time to make it less confusing as to which files should use markdown and which should use rdoc. Can you replace the backticks with `<tt>` instead?

```
$ echo '<tt>bin/rails stats</tt>' | rdoc --pipe

<p><code>bin/rails stats</code></p>
```

Reply:
> I think we only want to change docs to markdown entire frameworks at a time 

Ah, did not know that - but makes sense. Updated to `tt`-wrap.